config/database.yml

```diff
@@ -20,6 +20,10 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
+  host: db
```

Comment:
Let's set these to vars in .env, then create `.env.example-local, .env.example-docker` and set them up appropriately for each env.